lib/search/cubit/search_cubit.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:fluttercon/common/data/models/failure.dart';
+import 'package:fluttercon/common/repository/db_repository.dart';
+import 'package:fluttercon/search/cubit/search_state.dart';
+import 'package:fluttercon/search/models/search_result.dart';
+import 'package:injectable/injectable.dart';
+
+@injectable
+class SearchCubit extends Cubit<SearchState> {
+  SearchCubit(
+    this._dbRepository,
+  ) : super(const SearchState.initial());
+
+  final DBRepository _dbRepository;
```

Comment:
Check the signature of the `FetchIndividualOrganisersCubit` constructor to see how to inject the database layer uniformly.

Reply:
fixed